packages/pulp-ostree/pulp-ostree.spec

```diff
@@ -6,12 +6,12 @@
 
 # ---- archive related macros ----
 
-%define git_tag %{version}b1
+%define git_tag 1.4-release
```

Comment:
should this point to a tag instead of a branch?

Reply:
Generally, it shouldn't be changed.  IIRC @zjhuntin pushed this up in case he is unable to take it across the finish line, so one of us could take it over.

For the test builds, we use the branch name (it still works in the same way)

Reply:
gotcha.